README.md

```diff
@@ -1074,7 +1074,51 @@ RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
     && pk12util -d sql:/home/seluser/.pki/nssdb -i client_cert.p12 -W password_of_clent_cert # client certificate install
 ```
 This way the certificates will be installed and the node will start automatically as before.
+___
+
+## Alternative method: Add certificates to existing Selenium based images for browsers
+
+As an alternative, you can add youe certificate files to existing Selenium images. This practical example
+assumes you have a known image to use as a build image and have a way to publish new images to your local
+docker registry.
+
+This also assumes you are using a redhat based image as *build* _with_ your internal CA already added.
+(default location for rocky linux is /etc/pki/ca-trust/source/anchors/YOUR_CA.pem)
+
+(tested working 2023-05-17, based on images from Selenium 4.9.1 release)
```

Comment:
This is not really needed, we can always troubleshoot or tell people to use latest.

Reply:
ok